src/Catalyst.jl

```diff
@@ -42,6 +42,7 @@ import Base: (==), hash, size, getindex, setindex, isless, Sort.defalg, length,
 import MacroTools, Graphs
 import Graphs: DiGraph, SimpleGraph, SimpleDiGraph, vertices, edges, add_vertices!, nv, ne
 import DataStructures: OrderedDict, OrderedSet
+import LinearAlgebra.eigvals
```

Comment:
changed. Especially for a few of these niche ones, it might make sense to have some kind of comment on why they are needed/which feature uses them. I feel we are starting to accumulate quite a few dependencies, and it would probably make sense to keep track of which we have and why.